viewer/src/components/items/Table.js

```diff
@@ -67,6 +64,7 @@ const Table = ({ table, project, itemWidth, height, width }) => {
     }
 
     return (
+        <ItemContainer>
```

Comment:
Might want this inside the provider

Reply:
What do you mean by inside the provider?

Reply:
<Provider>
  <ItemContainer/>
</Provider>